images/plugin-compiler/test.zsh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env zsh
```

Comment:
Can it be just bash for better compatibility?

Reply:
tbh, I don't know how to implement `TRAPINT()` in bash. The script is just sugar over two straightforward commands.

Reply:
If it helps, think this will trap Enter, SIGINT or SIGTERM
`
trap '' EXIT
echo ' --- press ENTER to close --- '
read response
`